x-pack/legacy/plugins/ml/public/data_frame_analytics/common/analytics.ts

```diff
@@ -189,26 +199,30 @@ export const loadEvalData = async ({
   isTraining,
   index,
   dependentVariable,
+  resultsField,
+  predictionFieldName,
 }: {
   isTraining: boolean;
   index: string;
   dependentVariable: string;
+  resultsField: string;
+  predictionFieldName?: string;
 }) => {
   const results: LoadEvaluateResult = { success: false, eval: null, error: null };
+  const defaultPredictionField = `${dependentVariable}_prediction`;
```

Comment:
When testing this I noticed that it is possible to create a job without entering the `prediction_field_name`. The Create button should be disabled unless they have entered a `prediction_field_name`.

Reply:
fixed with: df6b17110ca1d397b89ad24e07746feb6db7f293

Reply:
The `prediction_field_name` is optional (defaults to whatever `<dependent_variable>_prediction`) - do you mean the `dependent_variable`? If so that was fixed in this commit : df6b17110ca1d397b89ad24e07746feb6db7f293